### ISSUE TITLE
Assembly too small for polishing?

### ISSUE THREAD

**Issue by JWDebler** [isovic/racon#174]
Hi,
I am trying to polish a mitochondrial assembly of just over 50.000 bp.

I keep getting this error though:

```
minimap2 AlKewell_mito.fasta ../nanopore/AlKewell.nanopore.all.fastq.gz > AlKewell.paf

racon -m 8 -x -6 -g -8 -w 500 ../nanopore/AlKewell.nanopore.all.fastq.gz AlKewell.paf AlKewell_mito.fasta > AlKewell_mito.racon.fasta

[racon::Polisher::initialize] loaded target sequences 0.000429 s
[racon::Polisher::initialize] loaded sequences 116.784445 s
[racon::Overlap::transmute] error: unequal lengths in target and overlap file for target AlKewell_mito!

```

I used the same command on the full genome assembly and that worked fine. Is my fragment too small? As it might be smaller than some of the contained reads.

**Comment by rvaser** [comment]
Hi Johannes,
there are no boundaries on sequence length in Racon. Can you please paste the output of the following commands:
```bash
head -n 1 AlKewell.paf
tail -n +2 AlKewell_mito.fasta | wc
```
Best regards,
Robert

**Comment by JWDebler** [comment]
```
head -n 1 AlKewell.paf
532abcbc-a198-494f-b540-cc61a136b2c5    723     15      710     -       AlKewell_mito   52512   8653    9351    504     704     60      tp:A:P  cm:i:76 s1:i:504        s2:i:0  dv:f:0.0373     rl:i:0

tail -n +2 AlKewell_mito.fasta | wc
    657     657   53826
```

**Comment by rvaser** [comment]
Are there carriage return (`\r`) characters in `AlKewell_mito.fasta`? Please run `head -n 2 AlKewell_mito.fasta`.

**Comment by JWDebler** [comment]
Wow, thanks. Yep, what a noob mistake. I exported the fasta from geneious which seems to have inserted invisible characters.

`dos2unix` fixed it. 

Is this a problem of minimap or racon parsing the fasta? I will notify the geneious developers, as I now remember running into a similar problem in the past after exporting a fasta.

Cheers!

**Comment by rvaser** [comment]
The problem is that the parser in Racon does not remove whitespaces if the sequence is in multiple lines, I will have to fix this :)

**Comment by rvaser** [comment]
Fixed from `1.4.20` (maybe even sooner).
